ydb/core/viewer/json_pipe_req.h

```diff
@@ -47,6 +47,7 @@ class TViewerPipeClient : public TActorBootstrapped<TViewerPipeClient> {
     NWilson::TSpan Span;
     IViewer* Viewer = nullptr;
     NMon::TEvHttpInfo::TPtr Event;
+    NHttp::TEvHttpProxy::TEvHttpIncomingRequest::TPtr HttpEvent;
```

Comment:
std::variant<> for this
or they can be both set at the same time?

Reply:
```
xenoxeno@mr-nvme-testing-01:/home-big/xenoxeno/work/ydb/ydb/core/viewer$ fgrep -rn "Event->" | wc -l
228
```

maybe later...